### PR TITLE
Desktopversion

### PR DESCRIPTION
**Pull request by George7h**
Add desktop styling

updated html due to responsive design not desirable.

### CODE REVIEW

File: HTML&CSS/Website.html [hunk 1]
```diff
@@ -133,41 +152,60 @@ <h2 class="connect"><a href="#">LET'S CONNECT</h2></a>
     <button class="resume" href="#">Get my resume</button>
```
Comment (AdnanOlarmmi) [inline HTML&CSS/Website.html:152]: - [ ] Your design is almost good, kindly fix the border radius of this button, it changes on hover,

image

It should only manifest the hover border radius on hover and on unable

File: HTML&CSS/Website.html [hunk 2]
```diff
@@ -133,41 +152,60 @@ <h2 class="connect"><a href="#">LET'S CONNECT</h2></a>
...
     </div>
 
-    <div class="containing">
-    <ul class="skill-list">
-    <ul class="languages">
-    <li class="wrap">
-        <h3 class="language-title">Languange</h3>
-        <img src="../HTML&CSS/icons/Disabled.png">
-    </li>
-
```
Comment (AdnanOlarmmi) [inline HTML&CSS/Website.html:143]: - [ ] Kindly add more spacing in between these containers to have a pixel perfect design, like you'll soon be doing in your next job.

- Your design
image

- Figma design
image

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @George7h, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

You're Almost there, funny one eyed alien

## Highlights
- Good UI design.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: HTML&CSS/Website.html (hunk 1 above)
Comment (George7h) [reply HTML&CSS/Website.html:152]: Thank you for the feedback.

File: HTML&CSS/Website.html (hunk 2 above)
Comment (George7h) [reply HTML&CSS/Website.html:143]: Thank you for your feedback. Adjustments have been made

**Review by Johnadibe — APPROVED** [review]
# Status: Approved ✔️
### Hi @George7h👋🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

LeonardoInGatsby

## Highlights
- Implemented all changes requested by the previous reviewer✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**